lib/govspeak.rb

```diff
@@ -305,7 +305,7 @@ def render_image(image)
       <<~BODY
 
         <div class="address"><div class="adr org fn"><p markdown="1">
-        #{body.lstrip.gsub(/\r?\n/, '<br />')}
+        #{body.strip.gsub(/ *\r?\n/, '<br />')}
```

Comment:
**question:** syntax coverage

Do we also need to strip trailing slashes, or are we confident no one ever used them (in a way that's now not wanted)?

Reply:
Ah right, two trailing backslashes have the same effect as two trailing spaces. https://kramdown.gettalong.org/quickref.html#paragraphs:~:text=using%20two%20spaces-,or%20two%20backslashes,-at%20the%20end

I've checked my local (a few weeks out-of-date) Whitehall database and can't find any uses of trailing slashes inside address blocks. (FWIW, I did see them used inside $C contacts on my travels, though.)

I think I've unintentionally broken this Kramdown feature (for address blocks) with my changes. It seems to remove some of the slashes but not others. I hope it isn't that `\r?` 🙈 I'll take a look, because we either want it to work or, as you suggested, we should fully remove them.

Reply:
It turned out that a few commits on this branch broke the trailing backslashes. It was easier to just strip them in the end rather than getting them working again.

But I have had to introduce that change in the same commit as another change. I couldn't figure out how to do either without breaking the other 😵